fabric-fluids-v1/src/main/java/net/fabricmc/fabric/api/fluids/v1/properties/NbtIdentifier.java

```diff
@@ -0,0 +1,38 @@
+package net.fabricmc.fabric.api.fluids.v1.properties;
+
+public final class NbtIdentifier {
+	public final int type;
+	public final String namespace;
+	public final String path;
+
+	/**
+	 * @deprecated for minecraft namespaces only!
+	 */
+	@Deprecated
+	@SuppressWarnings ("DeprecatedIsStillUsed")
```

Comment:
Please do not add suppressions like this. If the constructor is being actively used then it should not be deprecated. If you care this much about segregating the Minecraft namespace, make a factory method `NbtIdentifier.forMinecraft` (or similar) rather than a constructor.